apps/consent/env.ts

```diff
@@ -0,0 +1,17 @@
+import { createEnv } from "@t3-oss/env-nextjs";
+import { z } from "zod";
+
+export const env = createEnv({
+  server: {
+    HYDRA_ADMIN_URL: z.string().default("http://localhost:4445"),
+    AUTH_URL: z.string().default("http://localhost:4002"),
```

Comment:
- we should probably call it `CORE_AUTH_URL` to differentiate with hydra or kratos auth url
- I think the AUTH_URL should have the `/auth` suffix, ie: `http://localhost:4002/auth` so that we want to change where the auth is pointing to (ie: `auth.domain.com` because we are migrating the auth system to an independant server, then it will be less code to update.)